AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Campaigns/CampaignDetailQueryHandler.cs

```diff
@@ -82,5 +90,13 @@ public CampaignDetailQueryHandler(AllReadyContext context)
 
             return result;
         }
+
+        private CampaignDetailViewModel.CampaignManagerInviteStatus GetCampaignManagerInviteStatus(CampaignManagerInvite campaignManagerInvite)
+        {
+            if (campaignManagerInvite.IsAccepted) return CampaignDetailViewModel.CampaignManagerInviteStatus.Accepted;
+            else if (campaignManagerInvite.IsPending) return CampaignDetailViewModel.CampaignManagerInviteStatus.Pending;
+            else if (campaignManagerInvite.IsRejected) return CampaignDetailViewModel.CampaignManagerInviteStatus.Rejected;
+            else  return CampaignDetailViewModel.CampaignManagerInviteStatus.Revoked;
+        }
```

Comment:
> And to the second part, I think revoked would simply recall the message

how do you recall an email message? You can't. Once it's sent, it's sent.

Unless the way we're sending these invites is not through email, but some internal messaging functionality we intend to build?